Mathlib/Algebra/Group/Units/Equiv.lean

```diff
@@ -25,11 +26,17 @@ def toUnits [Group G] : G ≃* Gˣ where
 #align to_units toUnits
 #align to_add_units toAddUnits
 
+/-- The unit group of a unit group is equivalent to the same group. -/
+@[to_additive " The additive unit group of an additive unit group is equivalent to the same
+group. " ]
+def unitsTypeUnitsTypeEquivUnitsType [Monoid M] : Mˣˣ ≃* Mˣ := toUnits.symm
```

Comment:
```suggestion
def unitsUnitsEquivUnits [Monoid M] : Mˣˣ ≃* Mˣ := toUnits.symm
```
Why do you need this special case of `toUnits`?

Reply:
I was sort of thinking it would be useful as a simp-type thing I think? Like it's a thing about taking units being idempotent. But I take your point.